### ISSUE TITLE
addNmeaListener NullPointerException

### ISSUE THREAD

**Issue by j05u3** [Lyokone/flutterlocation#457]
**Describe the bug**
It is a crash

**Expected behavior**
Not a crash

**Tested on:**
 - Android, API 28 ZTE Blade V10 Vita
 - Android, API 30 Google Pixel 2

**Additional logs**

``` shell
Fatal Exception: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean android.location.LocationManager.addNmeaListener(android.location.OnNmeaMessageListener)' on a null object reference
       at com.lyokone.location.FlutterLocation$6.onSuccess(FlutterLocation.java:393)
       at com.lyokone.location.FlutterLocation$6.onSuccess(FlutterLocation.java:389)
       at com.google.android.gms.tasks.zzn.run(zzn.java:4)
       at android.os.Handler.handleCallback(Handler.java:873)
       at android.os.Handler.dispatchMessage(Handler.java:99)
       at android.os.Looper.loop(Looper.java:192)
       at android.app.ActivityThread.main(ActivityThread.java:7169)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:504)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
```

It happens affects more than 20% of the users in one of my apps. Also happens a lot in my testing device.

Maybe delete the line (in FlutterLocation.java)
```
locationManager = null
```
since it depends on the application context and not on the activity, so we can let it be non-null always?

**Comment by j05u3** [comment]
Now I see there is a PR about it already: https://github.com/Lyokone/flutterlocation/pull/447

**Comment by Lyokone** [comment]
Hi, this was merged and release as part of 3.2.1, you can reopen this issue if it happens again, thanks :)
